4.analyze_data/notebooks/UMAP/1.visualize-UMAP_plate3.ipynb

```diff
@@ -0,0 +1,314 @@
+{
```

Comment:
<p>Looks like this figure doesn't match the figure that has been added in this PR. Are you able to rerun this notebook so that the output looks the same as the PNG in this PR.</p><p> </p><p>The other figure looks to be the same, just not this one.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/CFReT_data/pull/23/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>